DragaliaAPI/DragaliaAPI/Services/Game/StoryService.cs

```diff
@@ -217,6 +222,17 @@ private async Task<List<AtgenBuildEventRewardEntityList>> ReadQuestStory(int sto
                 {
                     await fortRepository.AddToStorage((FortPlants)reward.Id, reward.Quantity, true);
                 }
+                else if (storyId == 1001009)
```

Comment:
Could you extract this to a named constant or add a comment explaining what it means?